src/main/java/com/systelab/seed/patient/control/RealtimePatientTracking.java

```diff
@@ -48,14 +47,8 @@ public void onMessage(String message, final Session session) {
 
         try (JsonGenerator generator = Json.createGenerator(writer)) {
             generator.writeStartObject().write("patientid", "papaaa").write("patientname", "asdsdasd").writeEnd();
-        }
-
-        String jsonValue = writer.toString();
-
-        try {
-            session.getBasicRemote().sendText(jsonValue);
+            session.getBasicRemote().sendText(writer.toString());
         } catch (IOException e) {
-            // TODO Auto-generated catch block
             e.printStackTrace();
```

Comment:
we shouldn't have  "e.printStackTrace()" in the code. Use the Logger instead or escalate the exception

Reply:
Solved